feature/bgp/policybase/otg_tests/nested_policies/nested_policies_test.go

```diff
@@ -0,0 +1,779 @@
+	netv61 := bgp6Peer1.V6Routes().Add().SetName("v6-bgpNet-dev1")
+	netv61.Addresses().Add().SetAddress(advertisedIPv61.address).SetPrefix(advertisedIPv61.prefix)
+
+	// configure eBGP on OTG port2
+	ipv42 := td.otgP2.Ethernets().Items()[0].Ipv4Addresses().Items()[0]
+	dev2BGP := td.otgP2.Bgp().SetRouterId(atePort2.IPv4)
+	bgp4Peer2 := dev2BGP.Ipv4Interfaces().Add().SetIpv4Name(ipv42.Name()).Peers().Add().SetName(td.otgP2.Name() + ".BGP4.peer")
+	bgp4Peer2.SetPeerAddress(dutPort2.IPv4).SetAsNumber(ateAS).SetAsType(gosnappi.BgpV4PeerAsType.EBGP)
+
+	ipv62 := td.otgP2.Ethernets().Items()[0].Ipv6Addresses().Items()[0]
+	bgp6Peer2 := dev2BGP.Ipv6Interfaces().Add().SetIpv6Name(ipv62.Name()).Peers().Add().SetName(td.otgP2.Name() + ".BGP6.peer")
+	bgp6Peer2.SetPeerAddress(dutPort2.IPv6).SetAsNumber(ateAS).SetAsType(gosnappi.BgpV6PeerAsType.EBGP)
+
+	// configure emulated network on ATE port2
+	netv42 := bgp4Peer2.V4Routes().Add().SetName("v4-bgpNet-dev2")
+	netv42.Addresses().Add().SetAddress(advertisedIPv42.address).SetPrefix(advertisedIPv42.prefix)
+	netv62 := bgp6Peer2.V6Routes().Add().SetName("v6-bgpNet-dev2")
+	netv62.Addresses().Add().SetAddress(advertisedIPv62.address).SetPrefix(advertisedIPv62.prefix)
+}
+
+// VerifyDUTBGPEstablished
+func (td *testData) verifyDUTBGPEstablished(t *testing.T) {
+	sp := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(td.dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).Bgp().NeighborAny().SessionState().State()
+	watch := gnmi.WatchAll(t, td.dut, sp, 2*time.Minute, func(val *ygnmi.Value[oc.E_Bgp_Neighbor_SessionState]) bool {
+		state, ok := val.Val()
+		if !ok || state != oc.Bgp_Neighbor_SessionState_ESTABLISHED {
+			return false
+		}
+		return true
+	})
+	if val, ok := watch.Await(t); !ok {
+		t.Fatalf("BGP sessions not established: got %v", val)
+	}
+	t.Log("DUT BGP sessions established")
+}
+
+// VerifyOTGBGPEstablished verifies on OTG BGP peer establishment
+func (td *testData) verifyOTGBGPEstablished(t *testing.T) {
+	sp := gnmi.OTG().BgpPeerAny().SessionState().State()
+	watch := gnmi.WatchAll(t, td.ate.OTG(), sp, 2*time.Minute, func(val *ygnmi.Value[otgtelemetry.E_BgpPeer_SessionState]) bool {
+		state, ok := val.Val()
+		if !ok || state != otgtelemetry.BgpPeer_SessionState_ESTABLISHED {
+			return false
+		}
+		return true
+	})
+	if val, ok := watch.Await(t); !ok {
+		t.Fatalf("BGP sessions not established: got %v", val)
+	}
+	t.Log("OTG BGP sessions established")
+}
+
+// Verify that the parent import policy (v4LPPolicy) has a child policy attached
+
+func (td *testData) verifyNestedImportPolicyAttachedv4(t *testing.T) {
+	rpDefPath := gnmi.OC().RoutingPolicy().PolicyDefinition(v4LPPolicy).State()
+	subResponse := gnmi.Get(t, td.dut, rpDefPath)
+	callPolicy := subResponse.GetStatement(v4LPStatement).Conditions.GetCallPolicy()
+	if callPolicy != v4PrefixPolicy {
+		t.Fatalf("Incorrect nested policy. Expected: %s, Got: %s", v4PrefixPolicy, callPolicy)
```

Comment:
here and below: got before want
https://google.github.io/styleguide/go/decisions#got-before-want
some additional guidance here about  formatting for test failure messages https://google.github.io/styleguide/go/decisions#level-of-detail